paperqa/agents/search.py

```diff
@@ -31,12 +32,15 @@
 )
 
 from paperqa.docs import Docs
-from paperqa.settings import MaybeSettings, Settings, get_settings
+from paperqa.settings import IndexSettings, get_settings
 from paperqa.types import DocDetails
 from paperqa.utils import ImpossibleParsingError, hexdigest
 
 from .models import SupportsPickle
 
+if TYPE_CHECKING:
```

Comment:
if `TYPE_CHECKING` is an import, won't it always be truthy?
(might be a silly question)

Reply:
No it's only `True` for type checkers (e.g. `mypy`), and is `False` at runtime

Read here: https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING